README.md

```diff
@@ -1 +1,22 @@
-# m2w64-htslib-build
\ No newline at end of file
+# m2w64-htslib-build
+
+The workflow [msys2-htslib-build.yml][] builds a statically linked htslib.dll
+under msys2. See the [releases][] to download the tarball. To build a different
```

Comment:
Is it worth defining what 'statically linked' means in this situation, since it's not entirely statically linked...

If decide a good idea to define, here's my first attempt...
*'statically linked' here implies static linkage to all needed msys2 libraries that are not referencing .DLLs that are provided by windows.

Reply:
Good point, I'll add your expanded definition to the README